website/www/site/layouts/index.html

```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
   </div>
 {{ end }}
 
+{{ define "playground-section" }}
+  <div class="body__contained body__section-nav">
+    <h1>Try Beam Playground (Beta)</h1>
```

Comment:
Can we take out (Beta) from the header and instead add (Beta) to the link with the Playground?

Reply:
I was told to remove this from the current pr, it's going to be implemented in another pr

Reply:
This one will be implemented via [[BEAM-14010](https://issues.apache.org/jira/browse/BEAM-14010)]